openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -167,10 +167,13 @@ def add_to_transaction(neo4j_entities, transaction):
     def get_command_last_run(course_key, graph):
         """
         This information is stored on the course node of a course in neo4j
-        :param course_key: a CourseKey
-        :param graph: a py2neo Graph
-        :return: a string representation of when this course was last dumped to
-         neo4j or None
+        Args:
+            course_key: a CourseKey
+            graph: a py2neo Graph
+
+        Returns: None, if there's no record of this command last being run, or
```

Comment:
nit: I'd reverse the order in your description (e.g., returns the datetime the command was last run as text, or None if there's no such record)